Apps/ValidationTests/Scripts/config.json

```diff
@@ -1,6 +1,73 @@
 {
     "root": "https://cdn.babylonjs.com",
     "tests": [
+        {
+            "title": "Visibility",
+            "playgroundId": "#PXC9CF#2",
+            "referenceImage": "visibility.png"
+        },
+        {
+            "title": "Edges",
+            "playgroundId": "#TYAHX#114",
+            "referenceImage": "edges.png"
+        },
+        {
+            "title": "Outline",
+            "playgroundId": "#10WJ5S#6",
+            "referenceImage": "outline.png"
+        },
+        {
+            "title": "Glow layer and LODs",
+            "playgroundId": "#UNS6ZV#2",
+            "referenceImage": "glowlayerandlods.png"
+        },
+        {
+            "title": "Nested BBG",
+            "playgroundId": "#ZG0C8B#1",
+            "renderCount": 10,
+            "referenceImage": "nested_BBG.png"
+        },
+        {
+            "title": "GUI3D SpherePanel",
+            "playgroundId": "#HB4C01#9",
+            "renderCount": 50,
+            "referenceImage": "spherepanel.png"
+        },
+        {
+            "title": "Kernel Blur",
+            "playgroundId": "#Y0WKT0#0",
+            "referenceImage": "KernelBlur.png"
+        },
+        {
+            "title": "Instanced Bones",
+            "playgroundId": "#2YLJ1L#2",
+            "referenceImage": "instancedBones.png"
+        },
+        {
+            "title": "GLTF Mesh Primitive Attribute",
+            "playgroundId": "#DS8AA7#27",
```

Comment:
This is not quite right. We are still using the old playground for glTF. I would prefer that we stay consistent. Also, how are these sorted? We are adding them to the top of the file right now. Maybe match the order in the source config?

Reply:
yes, many gltf tests this way in babylon.js VT.

Reply:
Babylon.js VT has support for sRGB, but Babylon Native does not yet. The new playground with the `__disableSRGBBuffers__` flag is not supported by Babylon Native yet. We should still be using `#DS8AA7#18` playground for this.